leader/undirected_ring/sync_higham_przytycka/sync_higham_przytycka.go

```diff
@@ -0,0 +1,413 @@
+package sync_higham_przytycka
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+
+	"github.com/krzysztof-turowski/distributed-framework/lib"
+)
+
+const DEBUG_ID_MOD = 1000
+const FIB64_UPPER_LIM = 93
+const LOG_PREFIX = "[%d:%01d] "
+
+type ID = int
+type direction = int
+type step = uint64
+
+const (
+	Left  direction = iota
+	Right direction = iota
+)
```

Comment:
It is sufficient to write
```
const (
	LEFT direction = iota
	RIGHT
)
```

Reply:
Please remove `= iota` on second and subsequent usages, as it should be inferred automatically by the compiler.